src/xdocs/config_misc.xml

```diff
@@ -28,7 +28,15 @@
         <p>
           Checks the style of array type definitions.  Some like Java style:
           <code>public static void main(String[] args)</code> and some like
-          C style: public static void main(String args[])
+          C style: <code>public static void main(String args[])</code>.
+        </p>
+        <p>
+          By default the Check enforces Java style.
+        </p>
+        <p>
+          This check strictly enforces only Java style for method return types
+          regardless of the value for 'javaStyle'. For example, <code>byte[] getData()}</code>.
```

Comment:
Sorry missed this one.
Remove extra `}`.

Reply:
Oh, I should have caught this myself - anyway, got it now - thanks again.